### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by samyakshah3008**
Here is the Product page for TripleS Store 

- Side Bar component

image

- Products component

image

- Also added Routers for landing page and product page 

- Please See that Filter functionality is left in product page 

You can live preview: https://spontaneous-cassata-6116f9.netlify.app/

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,23 +1,21 @@
...
-    <!--
+
+<head>
+  <meta charset="utf-8" />
+  <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <meta name="theme-color" content="#000000" />
+  <meta name="description" content="Web site created using create-react-app" />
+  <link rel="stylesheet" href="https://quirky-jang-ac862c.netlify.app/style.css">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css">
```
Comment (anuj-kumary) [inline public/index.html:11]: Give here some comment that which link is for what

File: src/components/Card.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+          <img className="card-image" src={item.imageURL} />
+          <i className="image-badge-wishlist far fa-heart"></i>
+        </div>
+
+        <div className="container2">
+          <div className="card-description"> {item.description} </div>
+          <div className="price">
+            <div className="discount-price"> {item.price} </div>
+            <div className="original-price">
+              <s> {item.originalPrice} </s>
```
Comment (anuj-kumary) [inline src/components/Card.jsx:18]: Good use of semantic Element got to know that  this tag `<s>` is used for strickthrough

**Review by Rohanmond — APPROVED** [review]
liked the way of structuring folders

File: src/pages/ProductPage.css [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+    box-shadow: 5px 5px 18px 5px var(--secondary);
+}
+
+
+
+.button2{
+    border: 2px solid var(--primary-color);
+    /* padding: 0.5rem 1rem;  */
+    font-size: 1rem;
+    /* transition: 0.5s; */
```
Comment (romabulani) [inline src/pages/ProductPage.css:150]: Remove the comments if not required

**Review by romabulani — APPROVED** [review]
- Liked the folder structure
- Learnt about `s` tag

Suggestions (if you have time)
- You can read about barrel expors and absolute paths, it will make the import paths shorter and will increase the readability of code.
- You can host the images on image hosting site like cloudinary it will increase the loading time

Also, its fine if you dont commit `package-lock.json` file on github, its not needed for hosting.

File: public/index.html [hunk 4]
```diff
@@ -1,23 +1,21 @@
...
+  <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+  <!--
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-    <!--
+  <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+  <!--
       Notice the use of %PUBLIC_URL% in the tags above.
```
Comment (hinanshisuthar) [inline public/index.html:19]: You can remove the comments that came along the installation of mockbee.

File: public/index.html [hunk 5]
```diff
@@ -26,12 +24,18 @@
...
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div style="z-index: 10000000000000000000000000000;" id="root"></div>
-    <!--
+  <title>React App</title>
+</head>
+
+<body>
+  <noscript>You need to enable JavaScript to run this app.</noscript>
+  <style>
```
Comment (hinanshisuthar) [inline public/index.html:32]: Consider external styling if possible

File: src/reducers/product-reducer.js [hunk 6]
```diff
@@ -0,0 +1,9 @@
+export const productReducer = (state,action) => {
+    switch (action.type) {
+        case "INITIAL_PRODUCT":
+        return {...state, products: [...action.payload]}
```
Comment (hinanshisuthar) [inline src/reducers/product-reducer.js:4]: Use prettier to format the file

**Review by hinanshisuthar — APPROVED** [review]
Good work samyak! Clean folder structure and. Suggested changes. Also, as roma said, know about barrel exports and host your images online. It'll save you much.

**Pull request merged**